### DISCUSSION TITLE
Html does not respond to pointer events

### DISCUSSION THREAD

**Discussion by jay-neb** [pmndrs/drei#560] [Q&A]
I've added a few Html components in my scene and I'm trying to add pointer events to them: `onPointerOver`, `onPointerOut` and `OnClick`, but they are not responding. I tried adding `pointerEvents: "all"` to the component's style, but it didn't help.

Am I missing something? Thank you.

**Comment by drcmda** [comment]
Maybe you have additional Dom layers up front, this is just css at this point. There are many examples in the R3f docs with clickable html components
